src/sass/form-updates.scss

```diff
@@ -0,0 +1,71 @@
+.has-feedback {
```

Comment:
I've used it from my previous projects. I also use bootstrap with a lot of unused rules 🙂